packages/drivers/routerlicious-driver/src/tokens.ts

```diff
@@ -32,11 +37,11 @@ export interface ITokenResponse {
 export interface ITokenProvider {
 	/**
 	 * Fetches the orderer token from host.
+	 *
 	 * @param tenantId - Tenant ID.
 	 * @param documentId - Optional. Document ID is only required for document-scoped requests.
 	 * @param refresh - Optional flag indicating whether token fetch must bypass local cache.
-	 * @returns TokenResponse object representing token value along with flag indicating
-	 * whether token came from cache.
+	 * Default: `false`.
```

Comment:
Curious why we're removing the `@returns` sections (here and below)?

Reply:
The details it was providing are already captured by the interface-level documentation of the return type, so they felt redundant to me. Happy to restore if others feel differently, though.